contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -7,6 +7,30 @@ should be applicable at later stages or for other projects using `klog` logging
 
 [Structured Logging KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
 
+## How to contribute
+
+### About the migration
+
+We organize migration component by component with time in between to collect feedback. 
+Migration of components that are not marked for it is on best effort and is expected to get delayed review or be rejected (example kubeadm).
+
+### What's the current status of migration 
+
+Kubelet was migrated, we are collecting feedback
+
+### How to get review
```

Comment:
It's weird to have "how to get review" section before "create pr requirements" as usually those steps will be done in reverse order. First you create PR then ask for review.

Let's create one section called "Sending a Structured Logging Migration PR"

Reply:
ok